### PR TITLE
she: Remove DisplayHandle

### PR DESCRIPTION
**Pull request by nanoslayer**
Remove DisplayHandle as it doesn't seem to be of much use.

### CODE REVIEW

File: src/main/main.cpp [hunk 1]
```diff
@@ -67,7 +66,7 @@ int app_main(int argc, char* argv[])
     MemLeak memleak;
     base::SystemConsole systemConsole;
     app::AppOptions options(argc, const_cast<const char**>(argv));
-    she::ScopedHandle<she::System> system(she::create_system());
+    she::System* system = she::create_system();
```
Comment (felipemanga) [inline src/main/main.cpp:69]: If this is no longer a ScopedHandle it won't be disposed in the end.
Use a unique_ptr/shared_ptr with a custom deleter instead?

Comment (nanoslayer) [reply src/main/main.cpp:69]: I guess we might want to keep ScopedHandle then.

**Comment by Zughy** [comment]
This also appears in `skia_display.h` and `.cpp`

**Comment by nanoslayer** [comment]
> This also appears in `skia_display.h` and `.cpp`

Fixed.

**Review by Zughy — APPROVED** [review]

**Pull request merged**